pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+"Consensus" here means no concrete objections have been raised: an accepted PHEP must be one that the entire community can at least live with. The PHEP authors must make a compelling case for the PHEP; objectors must provide meaningful concerns that could, in principle, be addressed by the authors. All are expected to discuss in good faith, while recognizing that good faith does not guarantee consensus can be reached.
+
+For a PHEP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed change. The change must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate PyHC projects unduly. Finally, a proposed enhancement must be "pythonic" in order to be accepted by the community. ("Pythonic" is an imprecise term; it may be defined as whatever is acceptable to the community. This logic is intentionally circular.)
+
+The editors must post announcements of PHEP resolution to the PyHC mailing list.
+
+Once a PHEP has been accepted, the reference implementation, if applicable, must be completed. When the reference implementation is complete and incorporated into a PyHC project's source code repository, the status will be changed to "Final". PHEPs which do not require a reference implementation move immediately to "Final" upon acceptance.
+
+If a PHEP requires additional design and interface feedback before committing to long term stability, it must remain in "Draft". Alternatively, its scope can be reduced (e.g. by deferring some features to later PHEPs).
+
+If no progress is being made on the PHEP, the editor should consult the author. The author may resume work on the PHEP, [transfer it to another author](transferring-phep-ownership), or withdraw it. If the author takes none of these actions, the editor may choose to bring the PHEP to the community for acceptance or rejection in its current form.
+
+If a PHEP fails to find consensus, the author may choose to modify it for reconsideration in the future. The PHEP may also be "Withdrawn" if the PHEP author themself decides that the PHEP is actually a bad idea, or has accepted that a competing proposal is a better alternative. A PHEP can also be "Rejected" if it fails to find consensus and the author is not willing or able to modify it. Perhaps after all is said and done it was not a good idea. It is still important to have a record of this fact.
+
+When a PHEP changes state, the PHEP preamble must be updated accordingly. In addition to updating the Status field, the Resolution header must be added with a direct link to the minutes of the telecons and meetings where a decision on the PHEP was made.
```

Comment:
I worry about making this process any more complex. My gut reaction is to go with option 1 or 2; we're probably good, nothing needed. Although option 2, capturing names of those affirming in meeting notes, seems easy enough to do that I could advocate for it. We're already linking to the meeting minutes in the Resolution header, so as long as people are okay with putting their names on record, saying who affirmed in the minutes would be nice. I think I'm talking myself into option 2.

Reply:
Keeping the process lightweight: it seems reasonable to say nothing here as we usually capture attendees in the minutes. Or could say "link to the minutes...was made. Those present who wish to note their acceptance should be recorded in the minutes."

Reply:
Yeah, capturing who voted on what feels like a Julie or me task, since we're the ones who compile meeting/telecon minutes. At the risk of me forgetting we wanted to do this, I'll say it's probably not necessary to mention here in PHEP-1.

Reply:
Taking door 1 then and recording this as a rejected idea.

Reply:
Record of rejected idea [here](https://github.com/jtniehof/standards/blob/0cd4f81ca4dd6b498606aadb3bbe451bd0031b9b/pheps/phep-0001.md?plain=1#L298) and resolving.